modules/mono/editor/script_templates/EditorScenePostImport/basic_import_script.cs

```diff
@@ -7,12 +7,12 @@
 [Tool]
 public partial class _CLASS_ : _BASE_
 {
-    public override Object _PostImport(Node scene)
+    public override Godot.Object _PostImport(Node scene)
     {
         // Modify the contents of the scene upon import. For example, setting up LODs:
-//      scene.GetNode<MeshInstance3D>("HighPolyMesh").DrawDistanceEnd = 5.0
-//      scene.GetNode<MeshInstance3D>("LowPolyMesh").DrawDistanceBegin = 5.0
-        return scene // Return the modified root node when you're done.
+        // scene.GetNode<MeshInstance3D>("HighPolyMesh").DrawDistanceEnd = 5.0;
+        // scene.GetNode<MeshInstance3D>("LowPolyMesh").DrawDistanceBegin = 5.0;
```

Comment:
In the previous PR there was a comment about putting the comment at the start of the line so it works with Godot's script editor but since C# users are more likely to use an external editor I thought it made more sense to keep them indented (this makes them work in VSCode, I haven't tested in other editors but I assume they are more likely to work like this).